ci-operator/config/opendatahub-io/kueue/opendatahub-io-kueue-test.yaml

```diff
@@ -0,0 +1,50 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-odh-kueue
+  steps:
+    cluster_profile: aws-opendatahub
+    env:
+      BASE_DOMAIN: openshift-ci-aws.rhaiseng.com
+      HYPERSHIFT_AWS_REGION: us-west-2
+      HYPERSHIFT_NODE_COUNT: "3"
+    test:
+    - as: e2e-test-odh-kueue
+      cli: latest
+      commands: |
+        oc apply --server-side -k "github.com/opendatahub-io/kueue//config/default?ref=test"
```

Comment:
Maybe we want to test the `rhoai` overlay, which is the one that's used in RHOAI.

Reply:
I don't understand what do you mean by `rhoai overlay`. Could you please elaborate ?
@astefanutti

Reply:
@ChughShilpa I mean `github.com/opendatahub-io/kueue/config/rhoai` instead.

Reply:
Okay, understood now.